api/src/domain/updateSidebarCards.test.ts

```diff
@@ -125,6 +126,85 @@ describe("updateRoadmapSidebarCards", () => {
     });
   });
 
+  describe("welcome card", () => {
+    it("adds the welcome-up-and-running card when operating phase is UP_AND_RUNNING_OWNING and removes the welcome card", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING_OWNING",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome-up-and-running");
+      expect(updatedUserData.preferences.visibleSidebarCards).not.toContain("welcome");
+    });
+
+    it("adds the welcome-up-and-running card when operating phase is UP_AND_RUNNING and removes the welcome card", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome-up-and-running");
+      expect(updatedUserData.preferences.visibleSidebarCards).not.toContain("welcome");
+    });
+
+    it("removes the welcome-up-and-running card and adds the welcome card back if the user reverts from UP_AND_RUNNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome-up-and-running");
+      expect(updatedUserData.preferences.visibleSidebarCards).not.toContain("welcome");
+
+      const revertedUserData = {
+        ...updatedUserData,
+        profileData: {
+          ...updatedUserData.profileData,
+          operatingPhase: "FORMED_AND_REGISTERED" as OperatingPhaseId,
+        },
+      };
+
+      const updatedRevertedUserData = updateSidebarCards(revertedUserData);
+      expect(updatedRevertedUserData.preferences.visibleSidebarCards).not.toContain("welcome-up-and-running");
+      expect(updatedRevertedUserData.preferences.visibleSidebarCards).toContain("welcome");
+    });
+
+    const operatingPhases = [
```

Comment:
if you type this array using `const operatingPhases: Operating Phase =`
then you don't need `as` later on